Resources/public/css/theme/views/buttons/general.css

```diff
@@ -0,0 +1,8 @@
+/**
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+.cof-btn--rounded {
```

Comment:
What do you think about creating `modules` folder and keeping there non-view related CSS code?